pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -1455,6 +1457,22 @@ public void getInfoFailed(ManagedLedgerException exception, Object ctx) {
                     return null;
                 });
 
+        // Schema store ledgers
```

Comment:
@liudezhi2098 I remember you have similar experience in debugging meta infos. Do you think this patch is good to go? I'm a bit uncertain due to it add a heavy operation on CLI command.

Reply:
> I'm a bit uncertain due to it add a heavy operation on CLI command.

heavy operation? CLI command is client tool and it just prints the output at client side which should not cause any heavy CPU/IP/Memroy pressure.